domains/novafox.json

```diff
@@ -4,6 +4,8 @@
         "email": "foxsdenyt@gmail.com"
     },
     "record": {
-        "CNAME": "foxsdenyt-github-io.onrender.com"
+        "ALIAS": "foxsdenyt-github-io.onrender.com",
```

Comment:
This isn't a valid record that you can use here

Reply:
ill have to swap it to an A record at "216.24.57.1", which cant be direct access and has to come from a domain, see: https://docs.render.com/configure-other-dns)